wear/features/waiting/build.gradle.kts

```diff
@@ -39,6 +39,8 @@ android {
 dependencies {
     implementation(project(":wear:core:resource"))
 
+    implementation("androidx.viewpager2:viewpager2:1.0.0")
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/bc838f5b9b1240f78879f529b60ac6af8967f499/wear/features/waiting/build.gradle.kts#L42"></span>A newer version of androidx.viewpager2:viewpager2 than 1.0.0 is available: 1.1.0</td>
    </tr>
  </tbody>
</table>